packages/stream-metadata/src/routes/spaceRefresh.ts

```diff
@@ -32,52 +30,21 @@ export async function spaceRefresh(request: FastifyRequest, reply: FastifyReply)
 
 	try {
 		const path = `/space/${spaceAddress}/image`
-		await createCloudfrontInvalidation({ path, logger })
 
-		await refreshOpenSea(spaceAddress)
-		logger.info({ path }, 'OpenSea cache invalidated')
+		await Promise.all([
+			createCloudfrontInvalidation({ path, logger }),
+			refreshOpenSea({ spaceAddress, logger }),
+		])
```

Comment:
maybe this should be all settled? but either way both requests go out so i guess it doesn't really matter